### PR TITLE
Fix windows test

### PR DESCRIPTION
**Pull request by ksimon1**
This patch updates kubevirt version in windows test to 0.11 (because of KUBEVIRT_MEMORY_SIZE variable), updates windows image download function (now it should not timeout because of download lock), removes restriction for vm memory size (now we can set more memory for node)

### CODE REVIEW

**Comment by ksimon1** [comment]
@MarSik, @fromanirh, @yanirq please review

File: generate-templates.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 - connection: local
   hosts: 127.0.0.1
+  gather_facts: no
```
Comment (MarSik) [inline generate-templates.yaml]: This should probably go to a separate patch as it applies to Travis as well. It is correct though.

Comment (ksimon1) [reply generate-templates.yaml]: done - https://github.com/kubevirt/common-templates/pull/45
